src/Simplex_tree/test/simplex_tree_remove_unit_test.cpp

```diff
@@ -122,7 +122,7 @@ BOOST_AUTO_TEST_CASE(remove_maximal_simplex) {
   std::clog << "st.upper_bound_dimension()=" << st.upper_bound_dimension() << std::endl;
   BOOST_CHECK(st.upper_bound_dimension() == 3);
 
-  // Check dimension calls lower_upper_bound_dimension to recompute dimension
+  // Check dimension calls upper_bound_dimension to recompute dimension
```

Comment:
This change seems wrong.

Reply:
I rolled back this bad find/replace on [7592ece](https://github.com/GUDHI/gudhi-devel/pull/837/commits/7592ece0cfd599359a9d3dd9375c316ef9440457)